### ISSUE TITLE
Unable to add user account

### ISSUE THREAD

**Issue by Kizmar** [microsoft/azuredatastudio#15203]
Issue Type: <b>Bug</b>

I need to add a different Linked account, but when I click on the add account icon, it keeps taking me to http://localhost:50107/signin?nonce=OUZ8a3H49noCzoH150EtBg==

This is in a Hyper-V VM... Visual Studio Code is doing to same thing to me when I try to log in to sync settings.

Side note: is there a domain I can switch localhost:port to in order to get this to work for now?

Azure Data Studio version: azuredatastudio 1.28.0 (13e362762762e5cb76a5c8afd2bc650f48c2d3d8, 2021-04-15T00:24:15.710Z)
OS version: Windows_NT x64 10.0.19042

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-10700K CPU @ 3.80GHz (6 x 3792)|
|GPU Status|2d_canvas: unavailable_software<br>flash_3d: disabled_software<br>flash_stage3d: disabled_software<br>flash_stage3d_baseline: disabled_software<br>gpu_compositing: disabled_software<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: disabled_off<br>protected_video_decode: disabled_off<br>rasterization: disabled_software<br>skia_renderer: disabled_off_ok<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: unavailable_software<br>webgl2: unavailable_software|
|Load (avg)|undefined|
|Memory (System)|16.00GB (10.10GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>Extensions: none

**Comment by cssuh** [comment]
Hi @Kizmar, can you try selecting the device code option and see if you can authenticate through this method?

**Comment by Kizmar** [comment]
@cssuh Last time I used Azure Data Studio (several days ago), I was able to log in when it popped into a browser. Today, it pulled up the "Linked accounts" right-panel where I click on one and hit "Close" (?). Then it says "Failed to get credential for account ...".

I removed the linked account in question and re-added it. It opened a browser window and I was able to authenticate. That right-panel UI is a little un-intuitive, but it seems to work I think?
